Hcb badges

kimani-ndegwa opened the pull request.

Commit:
Postgresql Integration and Test runs

kimobrian254 committed:
Circle CI

Circle CI Config file

kimobrian254 committed:
Update __init__.py

kimobrian254 committed:
MagicMock package

kimobrian254 committed:
Update requirements.txt

kimobrian254 committed:
Update __init__.py

kimobrian254 committed:
Update requirements.txt

kimobrian254 committed:
Update requirements.txt

kimobrian254 committed:
Update __init__.py

kimobrian254 committed:
Update __init__.py

kimobrian254 committed:
Update requirements.txt

kimani-ndegwa committed:
(Feature): Adds integration with code climate
(Chore) : Updates README.md file wrt above.

kimani-ndegwa committed:
(Chore): Updates .gitignore to resolve conflict

Merge branch 'develop' of https://github.com/andela/healthchecks_b

Conflicts:
	.gitignore

kimani-ndegwa committed:
(Chore): Puts badges on the same line.

kimani-ndegwa committed:
(Chore) Adds badge from coveralls.io

kimani-ndegwa committed:
(Chore)Adds Python version badge
(Chore)Adds License Badge

kimani-ndegwa committed:
(Chore) Removes License Info. badge

kimani-ndegwa committed:
(Chore) Creates and switches to new branch hcb-badges
(Chore) Aligns the badges on one line

kimani-ndegwa committed:
(Chore) Removes arbitrary coverage badge

Commit:
(Chore) Adds venv to gitignore

Commit:
Resolves conflict

Commit:
Resolves

iamlordaubrey commented on .gitignore:
```diff
@@ -4,6 +4,7 @@ __pycache__/
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+venv
```
Any reason why you're having two virtual environment folders in your application?

kimani-ndegwa replied:
Thats actually whitney who had **venv** as her virtual environment. I will let her know.

iamlordaubrey reviewed (changes requested):
I wouldn't like to clutter the .gitignore file with multiple virtual env folders. We should all have one called `hc-venv` (as per the readme).

iamlordaubrey commented on README.md:
```diff
@@ -1,5 +1,11 @@
+[![Code Climate](https://codeclimate.com/github/andela/healthchecks_b/badges/gpa.svg)](https://codeclimate.com/github/andela/healthchecks_b)
+[![Test Coverage](https://codeclimate.com/github/andela/healthchecks_b/badges/coverage.svg)](https://codeclimate.com/github/andela/healthchecks_b/coverage)
+[![Issue Count](https://codeclimate.com/github/andela/healthchecks_b/badges/issue_count.svg)](https://codeclimate.com/github/andela/healthchecks_b)
+[![PyPI](https://img.shields.io/pypi/pyversions/Django.svg?maxAge=2592000)]()
```
While you're it, I'm not sure the python badge is necessary. If you can defend it's necessity, by all means please do so. Else, let's remove it. LESS is MORE 😁

kimani-ndegwa replied:
Ok. Noted as well.

kimani-ndegwa committed:
(Chore) Removes PyPI badges
(Chore) Removes Venv from .gitignore

kimani-ndegwa committed:
(Chore) Resolves conflict
Merge branch 'hcb-badges' of https://github.com/andela/healthchecks_b into hcb-badges

Conflicts:
	.gitignore

iamlordaubrey commented on .gitignore:
```diff
@@ -4,6 +4,6 @@ __pycache__/
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+hc_venv
```
virtual env should be hc-venv not hc_venv. Please take note... a hyphen, not an underscore.

kimani-ndegwa committed:
(Chore) Change hc_venv to hc-venv
